app/models/user.rb

```diff
@@ -0,0 +1,40 @@
+require 'bcrypt'
+require 'dm-validations'
+
+
+class User
+  include DataMapper::Resource
+  include BCrypt
+
+
+  property :id,    Serial
+  property :name,  String
+  property :username, String, required: true, unique: true,
+  message:'Username invalid'
+  property :email,  String, required: true, unique: true,
+  message: "Email address invalid"
+  property :password , Text
+  attr_accessor :password_confirmation
+  validates_confirmation_of :password,
+  message: "Password does not match confirmation"
+  validates_format_of :email, :as => :email_address
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)